doc/README.txt

```diff
@@ -1,11 +1,45 @@
 maptlotlib documentation
 ========================
 
+
+Building the documentation
+--------------------------
+
+A list of dependencies can be found in ../doc-requirements.txt.
+
+All of these dependencies can be installed through pip::
+
+  pip install -r ../doc-requirements.txt
+
+or conda::
+
+  conda install sphinx numpydoc ipython mock colorspacious pillow
+
+To build the HTML documentation, type ``python make.py html`` in this
+directory. The top file of the results will be ./build/html/index.html
+
+**Note that Sphinx uses the installed version of the package to build the
+documentation**: matplotlib must be installed *before* the docs can be
+generated. Even if that is the case, one of the files needed to do this,
+'../lib/matplotlib/mpl-data/matplotlibrc', is not version controlled, but
+created when matplotlib is built. This means that the documentation cannot be
+generated immediately after checking out the source code, even if matplotlib
+is installed on your system: you will have to run ``python setup.py build``
+first.
```

Comment:
python setup.py install. And I had no problem yesterday. I tried on a fresh checkout, fresh installed.
Maybe my computer got tired of building matplotlib's documentation…